ci-operator/config/openshift/knative-client/openshift-knative-client-release-1.3.yaml

```diff
@@ -79,9 +79,6 @@ images:
       - destination_dir: .
         source_path: /go/bin/sidecarcontainer
   to: knative-client-test-sidecarcontainer
-promotion:
-  name: knative-v1.3.1
-  namespace: openshift
 resources:
```

Comment:
Why do you only remove promotion and don't remove the whole file? The branch release-1.3 is no longer there so CI config is not required, IMO. Otherwise this PR looks good.

Reply:
Well, I was hesitant to remove it completely, because practically the branch is still present due to being protected. I'll raise that with DPTP team to get rid off it. But let's remove CI config for it now.

Reply:
https://github.com/openshift/release/pull/30640/commits/037da3d71d454fa85cc2a0c9e7be84a1817a1c5d